### PR TITLE
docs(examples): example TRX transfer

### PR DESCRIPTION
**Pull request by marianogoldman**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: packages/platform-sdk-trx/src/services/transaction.ts [hunk 1]
```diff
@@ -51,10 +50,16 @@ export class TransactionService implements Contracts.TransactionService {
...
-			);
+			const pk: string = await new PrivateKey(this.#config).fromMnemonic(input.sign.mnemonic);
+			// const pk = new PrivateKey(this.#config).fromMnemonic(input.sign.mnemonic);
+			console.log("pk", pk, "tx", transaction);
+			// Code: https://github.com/tronprotocol/tronweb/blob/master/src/lib/trx.js#L650
+			// const response = await this.#connection.trx.sign(
+			// 	transaction,
+			// 	pk, // This one expects a callback, but not sure what it should do. Removing await from above doesn't work
+			// );
+			const response = await utils.crypto.signTransaction(pk, transaction);
```
Comment (marianogoldman) [inline packages/platform-sdk-trx/src/services/transaction.ts:61]: @faustbrian Couldn't get this to work using `this.#connection.trx.sign()`, just looked at their [code here](https://github.com/tronprotocol/tronweb/blob/master/src/lib/trx.js#L650).
Using this, I get it to sign it. It logs a reasonable output, with signature included. But then it fails to broadcast it, some rejected promised issue.
```
Error: Promise rejected with value: false
    at process.<anonymous> (/home/mariano/dev/ark/platform-sdk/examples/node_modules/.pnpm/hard-rejection@2.1.0/node_modules/hard-rejection/index.js:15:12)
    at process.emit (events.js:315:20)
    at processPromiseRejections (internal/process/promises.js:245:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
```
If you have a few mins, perhaps you understand what's going on, or I'll continue tomorrow.

Comment (faustbrian) [reply packages/platform-sdk-trx/src/services/transaction.ts:61]: Will take a look in a bit.

Comment (faustbrian) [reply packages/platform-sdk-trx/src/services/transaction.ts:54]:
```
{
  "code": "TRANSACTION_EXPIRATION_ERROR",
  "txid": "b194ecbda1119a87e9eb6a360240733058ed9a780edf33a15eae3de185897537",
  "message": "7472616e73616374696f6e2065787069726564"
}
```

Seems to get rejected based on expiration if you use the API directly.

Comment (faustbrian) [reply packages/platform-sdk-trx/src/services/transaction.ts]: @marianogoldman pushed a temporary fix for the expiration but the signature fails now.

```
{
  "code": "SIGERROR",
  "txid": "9e25a80a4faf33b6eb5dc3c936a0f2affcc9a83d4a6326790cad98038e35218f",
  "message": "76616c6964617465207369676e6174757265206572726f722064363033333964643939373564386362363932353634333531393765393933396462346233643837383135353665336363383664346538326466636636623138383932373161633366393230353263373532303230653738613562373633303334663566366638613530626634616239366661363166366261383231623135323030206973207369676e656420627920544e6e4c374d334e744c377067537654505479525a3975617671356255547051434320627574206974206973206e6f7420636f6e7461696e6564206f66207065726d697373696f6e2e"
}
```

Comment (faustbrian) [reply packages/platform-sdk-trx/src/services/transaction.ts]: All issues should be resolved by my last 2 merges to master.

**Pull request merged**